packages/x64dbg.dbgchild.vm/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,62 @@
+$ErrorActionPreference = 'Stop'
+Import-Module vm.common -Force -DisableNameChecking
+
+VM-Remove-PreviousZipPackage ${Env:chocolateyPackageFolder}
+
+try {
+    $toolSrcDir = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
+    $toolDir = Join-Path ${Env:RAW_TOOLS_DIR} 'x64dbg\release' -Resolve
+
+    $packageArgs = @{
+        packageName   = ${Env:ChocolateyPackageName}
+        unzipLocation = $toolSrcDir
+        url           = 'https://github.com/therealdreg/DbgChild/releases/download/beta10/DbgChild.Beta.10.zip'
+        checksum      = 'f17f588795d8f5f94d71335a8acfa58946bb03a94a5637be7f3e804c652ea2b4'
+        checksumType  = 'sha256'
+    }
+    Install-ChocolateyZipPackage @packageArgs
+    $toolSrcDir = Join-Path $toolSrcDir 'DbgChild Beta 10'
+    VM-Assert-Path $toolSrcDir
+
+    $archs = @("x32", "x64")
+    foreach ($arch in $archs) {
+        $archDstDir = Join-Path $toolDir "${arch}" -Resolve
+        $pluginDstDir = Join-Path $archDstDir 'plugins'
+        if (-Not (Test-Path $pluginDstDir -PathType Container)) {
+            New-Item -ItemType directory $pluginDstDir -Force -ea 0 | Out-Null
+        }
+        VM-Assert-Path $pluginDstDir
+
+        # Move 32/64-bit plugin DLL itself into the arch directory
+        $pluginSrcPath = Join-Path $toolSrcDir "release\${arch}\plugins" -Resolve
+        Get-ChildItem -Path $pluginSrcPath -File | Move-Item -Destination $pluginDstDir -Force -vb
+        
+        # Note that we don't simply move all children including directories, because we don't want to overwrite plugins
+
+        # Move all the other arch-specific files
+        $archSrcPath = Join-Path $toolSrcDir "release\${arch}" -Resolve
+        Get-ChildItem -Path $archSrcPath -File | Move-Item -Destination $archDstDir -Force -vb
+        if (-Not(Test-Path "${archDstDir}\CPIDS" -PathType Container)) {
```

Comment:
What would happen on an upgrade? The path may exist and then the move wouldn't happen.

Perhaps if the folder doesn't exists, create it. After the folder existence check, copy over the files from the source to the destination. This would allow the upgrade scenario to simply overwrite the pre-existing files.

Reply:
For this directory that behavior is OK. It's where child PIDs are stored while debugging. If it exists on upgrade, we don't need to do anything. 

I guess `Move-Item` is probably not the clearest operation to use here, since the source directory is actually empty. I'll replace with `New-Item`.